### PR TITLE
nicer paragraphing for the index page

### PR DESCRIPTION
**Pull request by robx**

### CODE REVIEW

File: web/index.go [hunk 1]
```diff
@@ -60,26 +60,27 @@ var (
...
+<a href="https://github.com/alicebob/verssion/">Full source</a> for issues and suggestions.
 </div>
-<br />
 
 <h2>Feed</h2>
-Make a feed which combines multiple projects in a single feed:<br />
-<a href="./curated/">Create new feed!</a><br />
+Make a feed which combines multiple projects in a single feed:
+<a href="./curated/">Create new feed!</a>
+<p>
```
Comment (alicebob) [inline web/index.go]: It's not closed?

File: web/index.go [hunk 2]
```diff
@@ -60,26 +60,27 @@ var (
...
+
 	{{- if .curated}}
-		<br />
-		Your recent feeds:<br />
+		Your recent feeds:
+		<ul>
 		{{- range $id, $cur := .curated}}
-			- <a href="{{$.base}}/curated/{{$id}}/">{{$cur.Title}}</a><br />
+			<li><a href="{{$.base}}/curated/{{$id}}/">{{$cur.Title}}</a><br /></li>
```
Comment (alicebob) [inline web/index.go]: `<li>` and `<br>` ?

Comment (robx) [reply web/index.go]: ah that was meant to go

File: web/index.go (hunk 1 above)
Comment (robx) [reply web/index.go]: It's not necessary, don't know what's better style? But like this it does the job of the previous `<br />` better without being more verbose...

**Comment by alicebob** [comment]
thanks!

**Pull request merged**